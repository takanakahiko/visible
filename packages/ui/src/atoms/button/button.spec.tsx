import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Button } from '.';

describe('Button', () => {
  it('matches snapshot', () => {
    const { container } = render(<Button />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
