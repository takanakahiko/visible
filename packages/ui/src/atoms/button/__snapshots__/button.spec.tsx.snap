// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders primary button 1`] = `
<button
  class="sc-bdVaJa emtJEX"
/>
`;

exports[`Button renders skeleton button 1`] = `
<button
  class="sc-bdVaJa hownKz"
/>
`;
